src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx

```diff
@@ -60,6 +62,7 @@ const CookiesPolicyBanner = ({
               width: 285,
               height: 48,
               padding: '14.5px 76px',
+              marginRight: notMobile ? 24 : 0,
```

Comment:
We should avoid this as much as possible use this approach and do a native solution instead:

You can do this:
```ts
const StyledButton = styled(CustomButton)({
  // your styled here...

  '& > div': {
    // ... your button text styles here...
  }

  [lightTheme.breakpoints.up('table_834')]: {
    marginRight: 24,
    // ...
  }
});
```

The previous snippet works because you can extend custom components that receive a `className` prop, behind scenes  `styled` transform you style in CSS style with a `className`

Reply:
you are right let's move to that solution